examples/android/RunAnywhereAI/app/src/main/java/com/runanywhere/runanywhereai/presentation/chat/ChatScreen.kt

```
@@ -1026,63 +911,70 @@ fun ThinkingToggle(
-                style = AppTypography.caption2,
-                color = MaterialTheme.colorScheme.onSurfaceVariant.copy(alpha = 0.5f),
-            )
-            Icon(
-                imageVector = Icons.Default.Lightbulb,
-                contentDescription = null,
-                modifier = Modifier.size(AppTypography.caption2.fontSize.value.dp),
-                tint = MaterialTheme.colorScheme.secondary,
-            )
+                // Generation time
+                Text(
+                    text = String.format("%.1fs", analytics.totalGenerationTime / 1000.0),
```

detekt (1.23.8)

[warning] String.format("%.1fs", analytics.totalGenerationTime / 1000.0) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)